packages/firestore/src/local/shared_client_state.ts

```diff
@@ -56,6 +58,8 @@ const QUERY_TARGET_KEY_PREFIX = 'firestore_targets';
 
 // The LocalStorage key that stores the primary tab's online state.
 const ONLINE_STATE_KEY = 'firestore_online_state';
+// The LocalStorage key that stores the last sequence number allocated.
+const SEQUENCE_NUMBER_KEY = 'firestore_sequence_number';
```

Comment:
If I am not mistaken, the `SEQUENCE_NUMBER_KEY` should include the persistence key prefix, so that it is only shared within a project? While you think about that, I'll figure out if I have to do the same thing for the ONLINE_STATE_KEY.

Reply:
Hmm, yeah, probably should be scoped to a project. I'll fix it.